src/uploaders.rs

```diff
@@ -82,6 +83,17 @@ impl Uploader {
         format!("readmes/{name}/{name}-{version}.html")
     }
 
+    /// Returns the internal path of an uploaded crate's index file.
+    fn index_path(name: &str) -> String {
```

Comment:
This should be replaced with a call to `relative_index_file` in `lib.rs`.

Reply:
Done. I had to add a new function, since we don't actually want a `Path` here (we need a string).